ui/src/components/App.js

```diff
@@ -58,7 +59,6 @@ const ProvidedRoutes = () => (
                   render={({ match }) => (
                     <>
                       <Header />
-                      {/* <ArrangerDashboard basename={match.url} /> */}
```

Comment:
I'd say we can just get rid of this entire `<Route path="/arranger" ... />` altogether, instead of just removing the `ArrangerDashboard` component usage 🙂